tensorflow/compiler/tf2tensorrt/convert/convert_graph.cc

```diff
@@ -717,7 +717,8 @@ Status CreateStaticEngine(const ConversionParams& params,
       trt_allocator.get(), /*calibrator=*/nullptr, &engine,
       info.use_calibration, params.use_implicit_batch,
       /*convert_successfully=*/nullptr, profile, info.engine_name,
-      /*use_explicit_precision=*/params.use_explicit_precision));
+      /*use_explicit_precision=*/params.use_explicit_precision,
+      const_cast<tensorflow::grappler::Cluster*>(params.cluster)));
```

Comment:
Because `params.cluster` is const while `TRTOptimizationPass::Optimize` takes a non-const pointer.
Making `params.const` non-const would be also an option.

Reply:
You are right, I refactored it as suggested in the latest commit